omnipaxos/src/sequence_paxos/mod.rs

```diff
@@ -237,9 +237,13 @@ where
                         .get_stopsign()
                         .expect("storage error while trying to read stopsign")
                     {
-                        if !ss.decided {
+                        let decided_idx = self
+                            .internal_storage
+                            .get_decided_idx()
+                            .expect("storage error while trying to read decided_idx");
+                        if !ss.decided(decided_idx) {
                             for follower in self.leader_state.get_promised_followers() {
-                                if !self.leader_state.follower_has_accepted_stopsign(follower) {
+                                if !self.leader_state.get_accepted_idx(follower) > ss.log_idx {
```

Comment:
This comparison looks a bit weird, maybe just do
```suggestion
                                if self.leader_state.get_accepted_idx(follower) < ss.log_idx {
```